src/main/java/baubles/common/items/ItemRing.java

```diff
@@ -55,14 +65,13 @@ public ItemStack onItemRightClick(ItemStack par1ItemStack, World par2World, Enti
 					break;
 				}
 		}
-
-		return par1ItemStack;	
+		return new ActionResult<ItemStack>(EnumActionResult.PASS, par1ItemStack);	
```

Comment:
You probably want SUCCESS here...

Or, return success instead of breaking the loop, and return pass here.

Either way, the way it is right now, putting a bauble on will always trigger your off hand item as well